kubernetes/samples/scripts/common/domain-template.yaml

```diff
@@ -47,6 +47,8 @@ spec:
       value: "%JAVA_OPTIONS%"
     - name: USER_MEM_ARGS
       value: "-XX:+UseContainerSupport -Djava.security.egd=file:/dev/./urandom "
+    - name: ISTIO_ENABLED
+      value: "%ISTIO_ENABLED%"
```

Comment:
We shouldn't be requiring users to set this value - the Operator should set it.  If we want to keep it here, maybe rename it ISTIO_ENABLED_EXPERIMENTAL or at least put a clear comment above the env var stating that it's experimental.